src/content/chainlink-automation/overview/supported-networks.mdx

```diff
@@ -50,8 +50,6 @@ To use Chainlink Automation on certain networks, you may need to conduct token t
 <CcipCommon callout="thirdPartyApps" />
 <ResourcesCallout callout="bridgeRisks" />
 
```

Comment:
@wentzeld Do you prefer to remove this bit entirely, or change the contact method for this to Discord?